--- a/include/trace/hooks/mmc.h
+++ b/include/trace/hooks/mmc.h
@@ -14,9 +14,7 @@
  * Following tracepoints are not exported in tracefs and provide a
  * mechanism for vendor modules to hook and extend functionality
  */
-DECLARE_HOOK(android_vh_mmc_sdio_pm_flag_set,
-	TP_PROTO(struct mmc_host *host),
-	TP_ARGS(host));
+#define trace_android_vh_mmc_sdio_pm_flag_set(host) do { (void)(host); } while (0)
 
 DECLARE_RESTRICTED_HOOK(android_rvh_mmc_suspend,
 	TP_PROTO(struct mmc_host *host),
